server/cmd/command.go

```diff
@@ -345,3 +335,41 @@ func exportedFields(command reflect.Value) []reflect.StructField {
 	}
 	return fields
 }
+
+// SplitCommandArgs splits str to args for command execution.
+func SplitCommandArgs(str string) []string {
+	var (
+		args          []string
+		arg           strings.Builder
+		quote         bool
+		previousSpace bool
+	)
+
+	for _, char := range str {
+		switch char {
+		case '"':
+			quote = !quote
+		case ' ':
+			if quote {
+				// writing part of the string.
+				arg.WriteRune(char)
+				continue
+			}
+			if previousSpace {
+				// trimming space.
+				continue
+			}
+			// end of the arg.
+			previousSpace = true
+			args = append(args, arg.String())
+			arg.Reset()
+		default:
+			previousSpace = false
+			arg.WriteRune(char)
+		}
+	}
```

Comment:
I'm not a huge fan of re-inventing an entire parser for this. My suggestion would be this: 
- Keep using the csv reader, merge the first value in the slice returned with the second value if `first value == "/"`
- Remove all empty strings from the slice returned, except if the value is last in the slice.